modules/convert/user_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package convert
+
+import (
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUser_ToUser(t *testing.T) {
+
+	user, err := models.GetUserByID(1)
+	assert.NoError(t, err)
+	assert.True(t, user.IsAdmin)
+
```

Comment:
suggest use ``AssertExistsAndLoadBean()``

Reply:
this is the original test ...
But I'll impruve it  ...